vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/grid/demo/GridDemo.java

```diff
@@ -741,7 +741,7 @@ private void createGridWithLazyLoading() {
         addCard("Lazy Loading", "Grid with lazy loading", grid);
     }
 
-    private void createGridWithCustomRowCountEstimate() {
+    private void createGridWithCustomItemCountEstimate() {
         // begin-source-example
         // source-example-heading: Faster Scrolling with Custom Row Count Estimate
```

Comment:
Still referring to "Row count" instead of "Item count" in many places in the file, please check properly

Reply:
Done